### PR TITLE
PR: Add blue border when the terminal is focused

### PR DESCRIPTION
**Pull request by steff456**
This PR adds a blue border when the widget is focused,

Focused:
image

Not Focused:
image

Note: I was unable to use Spyder's `FrameWebView ` class because the WebView and QFrame of spyder-terminal diverge greatly from that implementation.

### CODE REVIEW

**Review by ccordoba12 — COMMENTED** [review]
@steff456, I'm glad you were able to use my work here to make Spyder-terminal look better.

Besides my small comment, I'd like to know if it's possible for you to remove the outside gray border that is displayed around tabs now. I think that border comes from the tab widget in Qt.

**Comment by steff456** [comment]
Without the double border:
image

**Comment by ccordoba12** [comment]
> Without the double border:

@steff456, I think I didn't explain myself very well on my comment above. The problem is not the blue border, which is really useful to see that Spyder-terminal is the current pane with focus. The problem is the double gray border that is displayed now on the vertical left and right sides of the pane:

imagen

Could you revert your previous change and try to find a way to remove that border instead? Thanks!

**Comment by ccordoba12** [comment]
Sorry, I got confused. Please ignore my previous comment and just focus on the screenshot above.

**Comment by steff456** [comment]
image

File: spyder_terminal/widgets/main_widget.py [hunk 1]
```diff
@@ -9,9 +9,11 @@
 
 # Standard imports
 import os
-import sys
-import requests
 import os.path as osp
+import qstylizer
```
Comment (ccordoba12) [inline spyder_terminal/widgets/main_widget.py:13]: `qstylizer` is a third-party import, so please move it to the next section of imports.

File: spyder_terminal/widgets/terminalgui.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 # Standard library imports
 import json
 import os
+import qstylizer
```
Comment (ccordoba12) [inline spyder_terminal/widgets/terminalgui.py:12]: The same applies here, i.e. this is a third-party import.

File: spyder_terminal/widgets/main_widget.py [hunk 3]
```diff
@@ -9,9 +9,11 @@
...
+import requests
+import subprocess
```
Comment (ccordoba12) [inline spyder_terminal/widgets/main_widget.py:15]: The usage of `subprocess` was removed in PR #273. So please remove from here.

**Review by ccordoba12 — COMMENTED** [review]
Great work @steff456 removing that ugly double border! Last minor comments then this should be ready.

File: spyder_terminal/widgets/main_widget.py (hunk 3 above)
Comment (ccordoba12) [reply spyder_terminal/widgets/main_widget.py:14]: @steff456, you forgot to remove this import.

**Review by ccordoba12 — APPROVED** [review]
Looks good to me now, thanks @steff456!

**Pull request merged**